lua/fzf-lua/fzf.lua

```
@@ -279,6 +279,8 @@ function M.raw_fzf(contents, fzf_cli_args, opts)
       ["SHELL"] = shell_cmd[1],
       ["FZF_DEFAULT_COMMAND"] = FZF_DEFAULT_COMMAND,
       ["SKIM_DEFAULT_COMMAND"] = FZF_DEFAULT_COMMAND,
+      ["FZF_LUA_SERVER"] = vim.g.fzf_lua_server or vim.v.servername,
```

Not sure I understand what `vim.v.servername` is used here? Why is `vim.g.fzf_lua_server` ever `nil`?

---

I also use `vim.env.FZF_LUA_SERVER or vim.env.NVIM` in `shell_helper`
Just defensive programming, should I remove it? It's harmless though.

---

> I also use `vim.env.FZF_LUA_SERVER or vim.env.NVIM` in `shell_helper` Just defensive programming, should I remove it? It's harmless though.

No problem with being defensive but in some cases if we know with 100% certainty a certain var must exist IMHO it’s better to let this fail and we learn from the error what went wrong, otherwise we may get into a situation where things work but not quite right and we’ll have a hard time figuring out the “voodoo” issue.

Not sure about this one though, maybe this can workaround the `serverstart` issue with a bad XDG_RUNTIME_DIR (#716), or it might be better to let or fail as not having g:_fzf_lua_server` is critical enough to purposefully fail this?